content/en/database_monitoring/connect_dbm_and_apm.md

```diff
@@ -93,6 +93,9 @@ DD_VERSION=(application version)
 {{< tabs >}}
 {{% tab "Go" %}}
 
+<div class="alert alert-info">This documentation is for the Go Tracer v1.x. If you are looking for v2.x preview documentation, see the <a href="/database_monitoring/connect_dbm_and_apm-gov2">Correlate Database Monitoring and Traces</a> documentation.</div>
```

Comment:
Rather than adding an entirely new page which is largely copy-paste, can we use nested tabs here instead to separate between v1 and v2? There's precedent for doing this within [this page](https://docs-staging.datadoghq.com/apm-platform/apmlp-29/v2-docs/tracing/other_telemetry/connect_logs_and_traces/java?tab=log4j2) so it should be possible. Docs team can confirm